### DISCUSSION TITLE
How to toggle a switch

### DISCUSSION THREAD

**Discussion by mirkoarere** [dylandoamaral/uptime-card#153] [Q&A]
Hi, i'm tring to toggle a switch whet tap the icon or the bar but nothin appear, maybe i'm setting the code in a bad way. can anyone help me?

type: custom:uptime-card
entity: binary_sensor.nas
icon: mdi:nas
name: NAS
hours_to_show: 24
status_adaptive_color: true
color:
  icon: grey
  ko: ' #ff0000'
show:
  footer: false
  header: true
  title: true
  icon: true
  status: false
  timeline: true
  average: true
alias: {}
bar:
  amount: 24
  height: 20
update_interval: 30
clip: {}
blink: {}
init:
  animation: null
  duration: 1
title_adaptive_color: true
icon_adaptive_color: true
tap_action:
  toggle: switch.wol_nas

**Comment by dylandoamaral** [comment]
You have to do something like:
```
tap_action:
  action: toggle
  entity: switch.wol_nas
```
